src/pages/get-connected/index.tsx

```diff
@@ -36,6 +39,26 @@ export default function GetConnected() {
     { value: "entrepreneur", label: "Entrepreneur" },
   ];
 
+
+  const yesNoOptions = ["Yes", "No"];
+  const maritalStatusOption = ["Single", "In a Relationship", "Married", "Divorced", "Widowed"];
+  const occupationOption = ["Student", "Working Adult / Self-Employed", "Unemployed / Retired", "Others"];
+
+  const knowAboutUsOption = [
+    { value: "Friends / Family / Colleagues", label: "Friends / Family / Colleagues" },
+    { value: "Social Media Ads - Facebook / Instagram / Xiaohongshu", label: "Social Media Ads - Facebook / Instagram / Xiaohongshu" },
+    { value: "Others", label: "Others" },
+  
+  ]
+  const [occupation, setOccupation] = useState(occupationOption[0]);
+  const [occupationOthers, setOccupationOthers] = useState("");
+  const [remark, setRemark] = useState("");
+  const [remarkError, setRemarkError] = useState(false);
+
+  const [howToKnow, setHowToKnow] = useState(knowAboutUsOption[0]!.value);
```

Comment:
you should not need a `!` here to force `non-null assertions` as it does not appear as undefined in the first place.